### ISSUE TITLE
small cosmetic suggestion for cromwell_jobs()

### ISSUE THREAD

**Issue by jayoung** [getwilds/rcromwell#42]
hi there,

as I explore rcromwell, I can tell that every time I run `cromwell_jobs()` I'm going to want to do something like this every time:

```
cromwell_jobs() %>% print(width=Inf)
```

or even better, this
```
cromwell_jobs() %>% relocate(workflow_name, status, submission, start) %>% print(width=Inf) 
```

So my suggestion is to make `cromwell_jobs()` arrange the columns of its output tibble in a user-friendly order.

picky! sorry.

Janet

**Comment by jayoung** [comment]
another cosmetic-ish thing: how about making `cromwell_call()` emit its output as a tibble (rather than data.frame), to be consistent with the output of `cromwell_jobs()` ?     Either tibble or data.frame is good but consistency between functions in the same package is nice.

**Comment by tefirman** [comment]
Totally agree on the column order of `cromwell_jobs`, it would be much more convenient in the order we have it on the "Workflows Run" table of the Shiny app (see screenshot below). We can definitely fix that!

As for the `cromwell_call` output, I'm seeing that returning tibbles on my end. Is there a particular job you used that returned a data.frame (job ID, WDL script, etc)?

![image](https://github.com/user-attachments/assets/b2efd26c-00b7-4fd3-a6a7-79fc4b1270d4)

**Comment by sckott** [comment]
Thanks @jayoung 

Made fixes for these two functions in PR #46
